### ISSUE TITLE
deploying ah-resque-ui under AH 18.1.0 fails to initialize plugin 

### ISSUE THREAD

**Issue by MarkPareja** [actionhero/actionhero#1160]
# Name of Issue
Description of Issue

- ActionHero Version: `18.1.0`
- Node.js Version: `9.2.0`
- Operating System: `Ubuntu 17.10`

# Steps to reproduce your error
1:` npm install ah-resque-ui --save`
2: 'add plugin' -> config -> plugins.js
```js
exports['default'] = {
    plugins: (api) => {
        return {
            'ah-resque-ui': { path: __dirname + '/../node_modules/ah-resque-ui' }
        }
    }
}
```
3: `npm start`
throws: 
```
Error with initializer step: /home/mpareja/WebstormProjects/tagger/initializers/ah-resque-ui.js
TypeError: InitializerClass is not a constructor
    at initializerFiles.forEach (/home/mpareja/WebstormProjects/tagger/node_modules/actionhero/classes/process.js:110:23)
    at Array.forEach (<anonymous>)
    at Process.initialize (/home/mpareja/WebstormProjects/tagger/node_modules/actionhero/classes/process.js:99:22)
    at handleMethod (/home/mpareja/WebstormProjects/tagger/node_modules/actionhero/bin/actionhero:81:29)
    at Object.<anonymous> (/home/mpareja/WebstormProjects/tagger/node_modules/actionhero/bin/actionhero:149:3)
    at Module._compile (module.js:641:30)
    at Object.Module._extensions..js (module.js:652:10)
    at Module.load (module.js:560:32)
    at tryModuleLoad (module.js:503:12)
    at Function.Module._load (module.js:495:3)
    at Function.Module.runMain (module.js:682:10)
    at startup (bootstrap_node.js:191:16)
    at bootstrap_node.js:613:3
Cannot shut down actionhero, not running
```

**Comment by evantahler** [comment]
(formatted original bug)

**Comment by evantahler** [comment]
Try now! 

It looks like some of the later versions of ah-node-resque weren't published to NPM (v0.5+ is required for ActionHero v18).  They are now!

**Comment by evantahler** [comment]
Oh, and per the readme (https://github.com/actionhero/ah-resque-ui#setup-for-actionhero-v18) don't forget to add the config file as well.
